clarity/src/vm/tests/mod.rs

```diff
@@ -162,7 +162,16 @@ pub fn tl_env_factory() -> TopLevelMemoryEnvironmentGenerator {
 pub struct MemoryEnvironmentGenerator(MemoryBackingStore);
 impl MemoryEnvironmentGenerator {
     fn get_env(&mut self, epoch: StacksEpochId) -> OwnedEnvironment {
-        let mut owned_env = OwnedEnvironment::new(self.0.as_clarity_db(), epoch);
+        let mut db = self.0.as_clarity_db();
+        db.begin();
+        db.set_clarity_epoch_version(epoch).unwrap();
+        db.commit().unwrap();
+        if epoch >= StacksEpochId::Epoch30 {
+            db.begin();
+            db.set_tenure_height(1).unwrap();
+            db.commit().unwrap();
+        }
+        let mut owned_env = OwnedEnvironment::new(db, epoch);
```

Comment:
Should these changes have happened in the previous PR? I thought memory env tests were passing okay in that PR without this.

Reply:
Yes, they should have, but I believe the problem wasn't hit in the previous PR because of the special case handling of block 0.